sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsClient.java

```diff
@@ -365,6 +366,21 @@ public Void deleteModel(String modelId) {
         return digitalTwinsAsyncClient.deleteModelWithResponse(modelId, context).block();
     }
 
-    //TODO: Decommission Model APIs (waiting for Abhipsa's change to come in)
+    /**
+     * Decommissions a model.
+     * @param modelId The Id of the model to decommission.
+     * @return Void
+     */
+    public Void decommissionModel(String modelId) {
```

Comment:
Return type here should be the keyword `void`, not the `Void` class type.

Reply:
[The more you know.](https://stackoverflow.com/questions/10839042/what-is-the-difference-between-java-lang-void-and-void)